x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/danger_eui_context_menu_item.tsx

```diff
@@ -8,5 +8,5 @@ import styled from 'styled-components';
 import { EuiContextMenuItem } from '@elastic/eui';
 
 export const DangerEuiContextMenuItem = styled(EuiContextMenuItem)`
-  color: ${(props) => props.theme.eui.textColors.danger};
+  color: ${(props) => props.theme.eui.euiTextSubduedColor};
```

Comment:
Should it be `euiColorDangerText`?

Reply:
Nothing says danger quite like a subdued color...

Pushed a fix for this, and manually checked the other text color updates to make sure nothing else swapped.